### ISSUE TITLE
RF Spectrum A Issues

### ISSUE THREAD

**Issue by adkinsjd** [lab11/signpost#58]
- [x] R8 and C7 Labels are switched (which is bad if you want a working regulator)

**Comment by ppannuto** [comment]
- [ ] Check that caps are on `VDDCORE` not `VDDOUT`

**Comment by adkinsjd** [comment]
This isn't a sam4L...

**Comment by ppannuto** [comment]
Yeah.. sorry, got trigger happy, I'd already deleted that comment

**Comment by brghena** [comment]
- [ ] Add a ground debug pin (so that the other pins have something to be referenced against)
